src/coord/dist_aggregate.c

```diff
@@ -86,6 +86,17 @@ static void buildMRCommand(RedisModuleString **argv, int argc, int profileArgs,
   // Numeric responses are encoded as simple strings.
   array_append(tmparr, "_NUM_SSTRING");
 
+  int argOffset = 0;
+  // Preserve WITHCOUNT/WITHOUTCOUNT flag from the original command
+  argOffset  = RMUtil_ArgIndex("WITHCOUNT", argv + 3 + profileArgs, argc - 3 - profileArgs);
+  if (argOffset != -1) {
+    array_append(tmparr, "WITHCOUNT");
+  }
+  argOffset  = RMUtil_ArgIndex("WITHOUTCOUNT", argv + 3 + profileArgs, argc - 3 - profileArgs);
+  if (argOffset != -1) {
+    array_append(tmparr, "WITHOUTCOUNT");
+  }
```

Comment:
Currently these argss are not passed to the shard? (for `FT.SEARCH`)
How did `WITHOUTCOUNT` worked? (which is not the default)